spec/classes/vault_spec.rb

```diff
@@ -125,6 +126,21 @@
           }
         end
 
+        context 'when disable mlock' do
```

Comment:
please update to match the test intention

Reply:
done